packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/index.js

```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouter from '@uirouter/angularjs';
+import angularTranslate from 'angular-translate';
+import 'ovh-ui-angular';
+
+import component from './detail.component';
+import routing from './detail.routing';
+
+const moduleName = 'ovhTelecomPackHostedEmailDetail';
```

Comment:
From: `packages/manager/apps/telecom/src/app/telecom/pack/index.js`
We have the following AngularJS module name: `ovhManagerTelecomPack`

To keep consistency, you can name your module with the same prefix.

```suggestion
const moduleName = 'ovhManagerTelecomPackHostedEmailDetail';
```